source/build/ukfast.md

```diff
@@ -64,7 +66,6 @@ Below are the different facets that make up a UKFast solution, click on the link
       <h2>Hosted Desktop</h2>
       <ul>
           <li><a href="remotedesktop.html">Remote Desktop Services</a></li>
-          <li><a href="citrix.html">Citrix</a></li>
```

Comment:
How come you're removing `Citrix`?

Reply:
As we'd rather off via FastDesk instead of dedi Citrix, Ant has requested it be removed.